monitored-httpclient/src/test/java/com/github/nhenneaux/resilienthttpclient/monitoredclientpool/SingleIpHttpClientTest.java

```diff
@@ -165,8 +168,12 @@ void shouldntCallCheckHealthStatusIfHealthyIsTrue() {
             // Then
             assertSame(httpClient, singleIpHttpClient.getHttpClient());
             assertTrue(singleIpHttpClient.isHealthy());
-            verify(httpClient, times(1)).sendAsync(any(),any());
+            verify(httpClient, times(1)).sendAsync(any(), any());
             assertThat("failedResponseCount", singleIpHttpClient.getFailedResponseCount(), equalTo(0));
         }
     }
+
+    static Consumer<HttpRequest.Builder> getRequestTransformer() {
+        return builder -> builder.POST(HttpRequest.BodyPublishers.ofString("{\"request\":{\"transactionType\":\"ECHO_TEST\"}}"));
```

Comment:
Better to define a test value for new tests and use the default for existing tests